frontend-react/src/components/Table/Pagination.test.tsx

```diff
@@ -63,12 +58,12 @@ describe("Pagination", () => {
                 currentPageNum,
                 setSelectedPage: jest.fn(),
             };
-            renderWithBase(<Pagination {...props} />);
+            renderApp(<Pagination {...props} />);
 
             const list = screen.getByRole("list");
-            const { getAllByRole } = within(list);
-            const items = getAllByRole(ITEM_ROLE);
-            const itemContents = items.map((item) => item.textContent);
+            const itemContents = Array.from(list.children).map(
```

Comment:
Ah, I don't think this is exactly equivalent of using `within` -- `children` would be the direct descendants whereas querying with `within` would check for all descendants, no?

Not a huge deal -- it's completely fine if it works for this test, but I'd still err on the side of using `within` in case the we change the markup to have an extra layer between the `list` ancestor and the list items 🤷